Yank/analyze.py

```diff
@@ -604,9 +604,20 @@ def extract_trajectory(output_path, nc_path, state_index=None, replica_index=Non
     try:
         nc_file = netcdf.Dataset(nc_path, 'r')
 
+        # Extract topology and system serialization
```

Comment:
Would it make more sense to just store a bool in the NC file to get the ``isPeriodic`` flag? Because this seems like it will pull a number of resources to do. It might be okay for now since we should avoid trying to store EVERYTHING in the NC file when we can just as well check the properties from reloading the system.